core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -292,7 +292,7 @@
       PropertyType.TIMEDURATION,
       "The amount of time to wait after requesting a write-ahead log to be recovered"),
   MANAGER_WALOG_CLOSER_IMPLEMETATION("manager.walog.closer.implementation",
-      "org.apache.accumulo.server.master.recovery.HadoopLogCloser", PropertyType.CLASSNAME,
+      "org.apache.accumulo.server.manager.recovery.HadoopLogCloser", PropertyType.CLASSNAME,
```

Comment:
Since the old class name was user-facing as a default property value, should we keep a compatibility implementation for that one also?

Reply:
Yes. That's done in the latest commit (and I had missed a replication related property too--updated it and added a compatibility impl for it).